pipreqs/pipreqs.py

```diff
@@ -145,11 +178,35 @@ def get_all_imports(path, encoding=None, extra_ignore_dirs=None, follow_links=Tr
     return list(packages - data)
 
 
+def filter_ext(file_name, acceptable):
```

Comment:
guess we could improve this functions name to something more meaningful.

something like `def file_extension_is_allowed(file_name, allow_list)`

Reply:
Changed it to `file_ext_is_allowed` to avoid having too large names for functions